utils/fileutils.go

```diff
@@ -0,0 +1,39 @@
+package utils
+
+import (
+	"errors"
+	"fmt"
+	"os"
+)
+
+// PathExists returns true if the given path exists, and false if it does not.
+// If it cannot determine whether or not a path exists (e.g., because of a
+// permissions problem), it will log an error and exit.
+func PathExists(path string) (bool, error) {
+	_, err := os.Stat(path)
+
+	if err != nil {
+		if errors.Is(err, os.ErrNotExist) {
+			return false, nil
+		}
+		return false, fmt.Errorf("unable to check for %s: %w", path, err)
+	}
+
+	return true, nil
+}
+
+// PathIsDirectory returns true if the given path is a directory, and false if
+// it is not. If it cannot determine whether or not a path is a directory
+// (e.g., because of a permissions problem), it will log an error and exit.
+func PathIsDirectory(path string) (bool, error) {
```

Comment:
This doesn't look like it's used anywhere?

Reply:
It's not anymore. It was at one point. I left it here on the theory that it might end up useful, but I'll remove it. We can always bring it back later.